tests/test_chiral_utils.py

```diff
@@ -138,3 +139,51 @@ def test_find_chiral_bonds():
     mol = Chem.AddHs(Chem.MolFromSmiles(r"N(C)C(C)=O"))
     res = chiral_utils.find_chiral_bonds(mol)
     assert res == set([(0, 2)])
+
+
+def test_chiral_flip_check():
+
+    # test maps that can swap "chirality of methane"
+    mol_a = Chem.AddHs(Chem.MolFromSmiles("C"))
+    mol_b = Chem.AddHs(Chem.MolFromSmiles("C"))
+
+    AllChem.EmbedMolecule(mol_a, randomSeed=0)
+    AllChem.EmbedMolecule(mol_b, randomSeed=0)
```

Comment:
is the intent here that they have the same conformation? (may be better to just copy over the conformer for clarity).

Reply:
Do you have a preferred idiom for copying a conformer from one mol to another?

(First method I tried https://github.com/rdkit/rdkit/issues/5116 causes memory errors due to a low-priority object lifetime bug in rdkit, and it seemed simpler to create two independent mols than to copy conformers safely.)

Reply:
hm good point - i was going to suggest just embedding a pre-generated SDF perhaps, but maybe this isn't a big deal.

Reply:
I'm more inclined to just hard code an SDF here - since the conformer generation tool in the RDKit can vary between versions. Eg. rdkit 2022 generates an R version, but when we do an upgrade later on it instead generates an S version.

Reply:
Addressed in https://github.com/proteneer/timemachine/pull/908/commits/9eb5f255e8d16f9b1f15724bc6725a93baa4a00a